guide/02-api-overview/release-notes.ipynb

```diff
@@ -1,5 +1,196 @@
 {
```

Comment:
<p>Sanctions&nbsp;<code><a href="https://developers.arcgis.com/python/api-reference/arcgis.learn.html#classify-objects" target="_blank">classify_objects()</a></code>&nbsp;over&nbsp;<code><a href="https://developers.arcgis.com/python/api-reference/arcgis.learn.html#arcgis.learn.FeatureClassifier.categorize_features" target="_blank">FeatureClassifier.categorize_features()</a></code>&nbsp;for feature categorization. Maybe Priyanka can confirm this. Last time I checked, <a href="https://developers.arcgis.com/python/api-reference/arcgis.learn.html#arcgis.learn.FeatureClassifier.categorize_features" target="_blank">categorize_features()</a> is recommended to distinguish it from pixel classification. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/583/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>Correct! <code>classify_objects </code>should be used over<code> FeatureClassifier.categorize_features()</code> as catrgorize_features is deprecated.</p>